tests/test_camera.py

```diff
@@ -1,8 +1,93 @@
+import pytest
+
 from ppb import BaseSprite
 from ppb import Vector
 from ppb.camera import Camera
+from ppb.events import SceneStarted
+
+
+def test_camera_sets_height():
+    """Guarantee that the height is initially set dynamically."""
+
+    cam = Camera()
+
+    # Going to keep the default frame of 10 game units wide.
+    assert cam.width == 10
+    # NotSet will be a flag so we have a sentinel value
+    assert cam.height is None
```

Comment:
None is the sentinel value for this?

Reply:
I had it as a flag, but realized None is always going to be an invalid value for this use case.

That said. . . the rest of the engine expects those sentinel flags, so should probably do the same here.

Reply:
None is python standard for "there's no value here". To be honest, replacing DoNotRender with None is perfectly reasonable?

Reply:
If we go that route, we should replace DoNotRender with something like a Square flag. In that case, we have two separate sentinel values: one will be the canonical way to remove an object from rendering, and the other needs to be the canonical way to do a drawn primitive.

Probably a conversation to take to a different issue, since it's a problem to be solved when we handle the different rendering contexts (Specifically, Text) thought I think the Animation system gave us a pretty good hint in that direction.